v03_pipeline/lib/tasks/update_variant_annotations_table_with_new_samples.py

```diff
@@ -83,63 +73,82 @@ def read_annotation_dependencies(self):
             annotation_dependencies['gencode_mapping'] = hl.literal(
                 load_gencode(GENCODE_RELEASE, ''),
             )
-
         return annotation_dependencies
 
     def requires(self) -> list[luigi.Task]:
+        upstream_table_tasks = []
```

Comment:
You could maybe slot `UpdateVariantAnnotationsTableWithUpdatedReferenceDataset` here when initializing the list!